middleware.ts

```diff
@@ -0,0 +1,26 @@
+import { NextRequest, NextResponse } from 'next/server'
+
+//regex to check if there's an extension in the path, ie .jpg
+const PUBLIC_FILE = /\.(.*)$/
+
+export async function middleware(req: NextRequest) {
+  if (
+    req.nextUrl.pathname.startsWith('/_next') ||
+    req.nextUrl.pathname.includes('/api/') ||
+    PUBLIC_FILE.test(req.nextUrl.pathname)
+  ) {
+    return
+  }
+
+  if (req.nextUrl.locale === 'default' && !req.nextUrl.pathname.endsWith('/')) {
+    return NextResponse.redirect(new URL(`/en${req.nextUrl.pathname}`, req.url))
```

Comment:
In this case we don't have a language, maybe we want to send the back to the index so that it can be selected, rather than defaulting?? (This however does maintain current function to default to en)

Reply:
Suggesting that we never default to a language; so if one isn't specified we re-route to the index where the user is forced to make a selection.

Maybe I miss understood how the routing works; if on `/en/landing`, could the code just specify `/status` (link) would the router get me to `/en/status` without this?

Reply:
hmm okay; I think what I'm after isn't worth the nitpickyness; so I'll drop it :)